tower/src/util/boxed/sync.rs

```diff
@@ -39,6 +40,15 @@ impl<T, U, E> BoxService<T, U, E> {
         let inner = Box::new(Boxed { inner });
         BoxService { inner }
     }
+
+    #[allow(missing_docs)]
```

Comment:
Can we add a short doc comment for this? Just something like
```suggestion
    /// Returns a [`Layer`] for wrapping a [`Service`] in a `BoxService` middleware.
```